js/main.js

```diff
@@ -303,24 +310,13 @@ function updateLegendDataSource(dataSource) {
     $("#legend #dataSource").html('<a href="' + url + '">' + title + '</a>');
 }
 
-
-/*
- * Convert a city ID to a UI label.
- */
-function cityIdToLabel(s) {
-    return s.replace(/(^|\s|\-)([a-zA-ZäöüÄÖÜß])/g, function (x) {
-        return x.toUpperCase();
-    });
-}
-
-
 /*
  * Set the current city.
  *
  * `city` is the city ID.
  */
 function setCity(city) {
-    var filename = 'cities/' + city + '.json';
+    var filename = 'cities/data/' + city + '.json';
```

Comment:
@dirkschumacher I am against moving the market files since this will cause a lot of trouble with pull requests and existing branches of people who cloned the repository earlier. What's your opinion about this @torfsen?